src/main/webapp/app/course/tutorial-groups/shared/tutorial-group-sessions-table/tutorial-group-session-row/tutorial-group-session-row.component.html

```diff
@@ -27,6 +27,34 @@
             </span>
         </button>
     </td>
+    <td [class.table-danger]="isCancelled" class="attendance-column">
+        <div class="input-group" *ngIf="!isReadOnly; else readOnlyAttendance">
+            <input
```

Comment:
Should we add a label for accessibility?

Reply:
I think the label is given by the column name right?

Reply:
Added it in ec780ba673f95f1e802f77349285327a8023fa1a